common/app/views/fragments/commentBox.scala.html

```diff
@@ -21,13 +21,24 @@
         </div>
     </div>
 
+    <div class="d-comment-box__onboarding">
+        <h3>Welcome, you’re about to make your first comment!</h3>
```

Comment:
Very small suggestion.  We may add the user firstname to have a more personal message, like `Welcome John`.